extensions/ql-vscode/package.json

```diff
@@ -123,6 +123,20 @@
           "maximum": 1024,
           "description": "Number of threads for running queries."
         },
+        "codeQL.runningQueries.saveCache": {
```

Comment:
Probably add a scope of `"scope": "window"` to ensure that downloaded workspaces don't accidentally blow up someone's hard drive.